icomoon/style.css

```diff
@@ -0,0 +1,60 @@
+@font-face {
```

Comment:
Move all of these styles to `./styles/_icomoon.scss`. And delete this file.

Reply:
Moved all styles. That wich path should be specified in the line 
"src:  url("../icomoon/icomoon.woff") format("woff");"
or this piece of code is no needed?

Reply:
Deleted and moved. Now icommon works correctly